### PR TITLE
Desktop Portfolio PR

### PR DESCRIPTION
**Pull request by Javed-Shahbaz**
## Dear Reviewer!
   - I completed the project using HTML & CSS best practices

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
+            I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
           <button class="blackButton">
              Start collaboration
           </button>
```
Comment (bobb-Rob) [inline index.html]: - [ ] This section does not match the Figma design. Notice the point marked with the red circle. The button there should be within the dark-blue background. Not just the button, please ensure the correct margin, position and layout are the same with the Figma design. Please compare this image below 👇🏾 

Your design
image

Figma Design
image

File: index.html [hunk 2]
```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
```
Comment (bobb-Rob) [inline index.html]: - [ ] Please Ensure that this section matches the Figma Design 👇🏾 
   Consider the following: 
   - The font size and font-family
   - Spacing - margin/padding
   - The `see project` button is touch the bottom edge.

Your Design
image

Figma Design
image

File: index.html [hunk 3]
```diff
@@ -32,7 +32,8 @@
...
+            I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
           <button class="blackButton">
```
Comment (bobb-Rob) [inline index.html:38]: - [ ] The about section too has some isue to fix. Please see the image below 👇🏾 

- The section highlighted with a red square and letter A is missing some illustration
- The button - `get my CV` does not match the Figma design. Please fix.

Your design
image

Figma Design
image

File: index.html [hunk 4]
```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
+            I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
```
Comment (bobb-Rob) [inline index.html]: - [ ] The Footer social media icons are not well aligned, please fix this

Your design
image

Figma design
image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Javed-Shahbaz, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply index.html:38]: Fixed.

File: index.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Javed-Shahbaz 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Good README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**